worlds/animal_well/client.py

```diff
@@ -170,6 +215,60 @@ def on_package(self, cmd: str, args: dict):
         if cmd == "Connected":
             self.slot_data = args.get("slot_data", {})
 
+        try:
+            def parseIds(data):
+                for item in data:
+                    if 'type' in item and 'text' in item:
+                        if item['type'] == 'player_id':
+                            item['text'] = self.player_names[int(item['text'])]
+                        elif item['type'] == 'item_id':
+                            item['text'] = item_id_to_name[int(item['text'])]
+                        elif item['type'] == 'location_id':
+                            item['text'] = location_id_to_name[int(item['text'])]
+
+                return data
+
+            if cmd == "PrintJSON":
+                msgType = args.get("type")
+                if msgType == "Chat":
+                    display_to_client(self, "".join(o['text'] for o in parseIds(args.get("data"))))
+                elif msgType == "Hint":
+                    text = "".join(o['text'] for o in parseIds(args.get("data")))
+                    display_to_client(self, text)
+                elif msgType == "Join":
+                    display_to_client(self, args.get('data')[0]['text'])
+                elif msgType == "CommandResult":
+                    display_to_client(self, args.get('data')[0]['text'])
+                elif msgType == "Tutorial":
+                    pass
+                elif msgType == "ItemSend":
+                    data = parseIds(args.get('data'))
+                    text = "".join(o['text'] for o in data).replace(')', '').replace('(', 'at ')
+                    display_to_client(self, text)
+                elif msgType == "Countdown":
+                    # {'cmd': 'PrintJSON', 'data': [{'text': '[Server]: GO'}], 'type': 'Countdown', 'countdown': 0}
+                    text = "".join(o['text'] for o in args.get('data'))
+                    display_to_client(self, text)
+                else:
+                    logger.info("Unhandled type of PrintJSON: [{}]: {}".format(msgType, args))
+            elif cmd == "Retrieved":
+                logger.info(f"[{cmd}]: {args}")
+            elif cmd == "ReceivedItems":
+                items = args.get("items")
+            elif cmd == "RoomUpdate":
+                checked_locations = args.get("checked_locations")
+            elif cmd == "Bounced":
+                pass
```

Comment:
```suggestion
            elif cmd == "Bounced":
                pass
            elif cmd == "None":
                display_to_client(self, args.get("data")[0]["text"])
```

For some reason, sending an item to a game using `/send` from the server console has the "None" type, which is weird since it's not in the network docs. So probably best just to display it? Might need to be changed later if None is used for more types of things.

Reply:
Added